partiql-ast/src/main/java/org/partiql/ast/v1/Ast.kt

```diff
@@ -50,6 +51,7 @@ import org.partiql.value.PartiQLValue
 import org.partiql.value.PartiQLValueExperimental
 
 // TODO docs for all factory methods and move to Kotlin sources
+//  Also consider defaults for nullable. Need to look more into backwards compatibility.
```

Comment:
(self-review): when converting the `SqlDialectTest` over to use the v1 AST, it was somewhat annoying to not have defaults for all of the nullable fields. Deferring adding defaults to a different PR since we'll need to examine backwards compatibility of defaults a bit further.

Reply:
ack. https://github.com/partiql/partiql-lang-kotlin/issues/1640